README.rst

```diff
@@ -0,0 +1,24 @@
+=====
+Django data access log
+=====
+
+Django data access log is a simple Django app to log the access into a JRF viewset.
+
+
+Quick start
+-----------
+
+1. Add "data_access_log" to your INSTALLED_APPS setting like this::
+
+    INSTALLED_APPS = [
+        ...
+        'data_access_log',
+    ]
+
+2. Run `python manage.py migrate` to create the models.
+
+3. Add the `@` decorator to any viewset you'd like to log::
+    class MyLoggedViewSet(ModelViewSet):
```

Comment:
include @data_access_log in the example code

Reply:
This is modified in the following commit

Reply:
ok, maybe next time include the full/right version here.